devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1740,6 +1744,7 @@ bool WholeBodyDynamicsDevice::attachAllFTs(const PolyDriverList& p)
         return false;
     }
 
+    yDebug() << "Elobaid: remappedMASInterfaces= " << remappedMASInterfaces.ftMultiSensors->getNrOfSixAxisForceTorqueSensors();
```

Comment:
Probably this can be deleted?

Reply:
Ah yes!, done in https://github.com/robotology/whole-body-estimators/pull/167/commits/fca9e1829ca5c403bd4c87a0a5302466c39c4da7